web/vibe/web/common.d

```diff
@@ -463,11 +463,14 @@ package struct WebParamAttribute {
 	string field;
 }
 
+package(vibe.web) enum bodyParamWholeName = "bodyParamWhole";
```

Comment:
What about using an empty string here and asserting in the two-argument `@bodyParam` overload that the field name is always non-empty? That would eliminate any possibility for name clashes. I'd also tend to encapsulate the "whole body" logic into `WebParamAttribute` (e.g. `WebParamAttribute.isAllInOne` or similar, which performs the name comparison) - thinking about it, it may also make sense to extend this to query string parameters later on.